File: RailwationBackEnd/Domain/Entities/City.cs
namespace Domain.Entities;

public class City: BaseEntity
{
    public string Name { get; set; } = string.Empty;

    public Guid CountryId { get; set; }
    public Country Country { get; set; }

    public ICollection<CityConnection> CityConnections { get; set; }

}

File: RailwationClient/src/app/dto/City.ts
import { CityConnection } from "./CityConnection";
import { Country } from "./Country";

export interface City {
    id: string,
    name: string,
    countryId: string,
    country: Country,
    fromCityConnections: CityConnection[],
    toCityConnections: CityConnection[],
}